LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Dan Urbanowicz
```

Comment:
This line needs to be updated to match the master branch

Reply:
Updated.